_posts/0000-01-02-intanrizqi.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![doggo and knife](https://user-images.githubusercontent.com/85548886/121177690-e8fd8100-c887-11eb-9cc9-9472beddd900.jpeg)